discovery-infra/test_infra/helper_classes/cluster.py

```diff
@@ -18,24 +18,34 @@
 class Cluster:
 
     def __init__(self, api_client, cluster_name=None, additional_ntp_source=None,
-                 openshift_version="4.6", cluster_id=None):
+                 openshift_version="4.6", cluster_id=None, user_managed_networking=False, high_availability_mode=consts.HighAvailabilityMode.FULL):
         self.api_client = api_client
 
+        self._high_availability_mode = high_availability_mode
         if cluster_id:
             self.id = cluster_id
         else:
-            self.id = self._create(cluster_name or "test-infra-cluster", additional_ntp_source, openshift_version).id
+            cluster_name = cluster_name or env_variables.get('cluster_name', "test-infra-cluster")
+            self.id = self._create(cluster_name, additional_ntp_source, openshift_version,
+                                   user_managed_networking=user_managed_networking, high_availability_mode=high_availability_mode).id
             self.name = cluster_name
 
-    def _create(self, cluster_name, additional_ntp_source, openshift_version):
+    def _create(self,
+                cluster_name,
+                additional_ntp_source,
+                openshift_version,
+                user_managed_networking=False,
+                high_availability_mode=consts.HighAvailabilityMode.FULL):
```

Comment:
const in in init, why to have those?

Reply:
Assuming we always need these values, then there must be default values for them.   What's wrong with it?